scalding-serialization/src/test/scala/com/twitter/scalding/serialization/macros/MacroOrderingProperties.scala

```diff
@@ -510,7 +512,7 @@ class MacroOrderingProperties
   }
 
   test("Test out ByteBuffer") {
-    BinaryOrdering.ordSer[ByteBuffer]
+    implicitly[OrderedSerialization[ByteBuffer]]
```

Comment:
It'd be nice to have a test confirming that an explicit instance doesn't get overridden by the `DefaultOrderedSerialization` import.

Reply:
Done, did a test with a companion object defined ordered serialization not being overridden